WISE-Community/WISE-Client
# feat(Authoring): Add branch route with basic params

## Changes
- Added "Branch off this step" option in the add step button menu
- Created "Create Branch" route
- Create Branch route allows choosing
  - Number of Branch Paths
  - How to Choose Among Paths
  - Choose Step
  - Choose Item

Note: The individual branch path parameters and the merge step parameter will be added later

## Test
- Make sure the "Branch off this step" button brings you to the "Create Branch" route
- Make sure "Number of Branch Paths" is only valid if it is 2 or more
- If "Score" or "Choice Chosen" are selected, make sure the "Choose Step" and "Choose Item" drop downs show up
- If "Score" is selected, make sure you can only select steps and components that can be scored
- If "Choice Chosen" is selected, make sure you can only select steps and components that are Multiple Choice
- If "Score" or "Choice Chosen" are selected and there is only one component that can be selected, automatically select it
- If you switch from "Score" to "Choice Chosen", and the previously selected component was not a Multiple Choice, unselect it
- When you click the "Submit" button, it should alert the parameters for testing purposes

#1745
